egs/librispeech/s5/local/prepare_dict.sh

```diff
@@ -72,7 +72,7 @@ if [ $stage -le 1 ]; then
   auto_lexicon_prefix="$g2p_dir/lexicon_autogen"
 
   mkdir -p $g2p_dir/log
-  auto_vocab_splits=$(eval "echo $auto_vocab_prefix.{$(seq -s',' $nj)}")
+  auto_vocab_splits=$(for n in $(seq $nj); do echo $auto_vocab_prefix.$n; done)
```

Comment:
also this... revert anything that doesn't have to do with brace expansion.  the bug we are trying to fix is with brace expansion.